newrelic/resource_newrelic_cloud_gcp_link_account.go

```diff
@@ -0,0 +1,148 @@
+package newrelic
+
+import (
+	"context"
+	"strconv"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/newrelic/newrelic-client-go/pkg/cloud"
+)
+
+func resourceNewRelicCloudGcpLinkAccount() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceNewRelicCloudGcpLinkAccountCreate,
+		ReadContext:   resourceNewRelicCloudGcpLinkAccountRead,
+		UpdateContext: resourceNewRelicCloudGcpLinkAccountUpdate,
+		DeleteContext: resourceNewRelicCloudGcpLinkAccountDelete,
+		Schema: map[string]*schema.Schema{
```

Comment:
Missing accountId. Customer might want to select a different NR account. Example of code here: https://github.com/newrelic/terraform-provider-newrelic/pull/1608/files#diff-e7f2180b00f9dc0400207dc507066a63d1b515cee7da2daf4acabd59f12d2ca5R24

Reply:
Added accountId to select the different NR accounts